diff --git a/node_modules/@kodadot1/minipfs/dist/index.cjs b/node_modules/@kodadot1/minipfs/dist/index.cjs
index 1cd40e9..f045cf3 100644
--- a/node_modules/@kodadot1/minipfs/dist/index.cjs
+++ b/node_modules/@kodadot1/minipfs/dist/index.cjs
@@ -21,7 +21,7 @@ const ipfsProviders = {
   cloudflare: "https://cloudflare-ipfs.com",
   ipfs: "https://ipfs.io",
   dweb: "https://dweb.link",
-  kodadot: "https://kodadot.mypinata.cloud",
+  kodadot: "https://cdn.preschian.xyz",
   rmrk: "https://rmrk.mypinata.cloud",
   fleek: "https://ipfs.fleek.co",
   nftstorage: "https://nftstorage.link",
diff --git a/node_modules/@kodadot1/minipfs/dist/index.mjs b/node_modules/@kodadot1/minipfs/dist/index.mjs
index 613a632..f35c140 100644
--- a/node_modules/@kodadot1/minipfs/dist/index.mjs
+++ b/node_modules/@kodadot1/minipfs/dist/index.mjs
@@ -19,7 +19,7 @@ const ipfsProviders = {
   cloudflare: "https://cloudflare-ipfs.com",
   ipfs: "https://ipfs.io",
   dweb: "https://dweb.link",
-  kodadot: "https://kodadot.mypinata.cloud",
+  kodadot: "https://cdn.preschian.xyz",
   rmrk: "https://rmrk.mypinata.cloud",
   fleek: "https://ipfs.fleek.co",
   nftstorage: "https://nftstorage.link",
